fbg-server/src/chat/chat.service.ts

```diff
@@ -74,13 +80,26 @@ export class ChatService {
   private async checkUserInMatch(userId: number, matchId: string) {
     const matchEntity = await this.matchService.getMatchEntity(matchId);
     const membership = matchEntity.playerMemberships.find(
-        (m) => m.user.id === userId,
-      );
+      (m) => m.user.id === userId,
+    );
     if (!membership) {
       throw new HttpException(
         `You need to be a player in this match in order to send messages.`,
         HttpStatus.BAD_REQUEST,
       );
     }
   }
+
+  private async checkChannelIsPublic(channelType: ChannelType, channelId: string) {
+    let roomEntity: RoomEntity;
+    if (channelType === 'match') {
+      const matchEntity = await this.matchService.getMatchEntity(channelId);
```

Comment:
We have to double check that the match is populated from the room in this SQL query